src/test/java/com/evanlennick/retry4j/CallExecutorBuilderTest.java

```diff
@@ -0,0 +1,81 @@
+package com.evanlennick.retry4j;
+
+import com.evanlennick.retry4j.config.RetryConfig;
+import com.evanlennick.retry4j.config.RetryConfigBuilder;
+import com.evanlennick.retry4j.listener.RetryListener;
+import org.testng.annotations.Test;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+import static org.testng.AssertJUnit.assertEquals;
+
+public class CallExecutorBuilderTest {
+
+  private RetryListener afterFailedTryListener = status -> {};
+  private RetryListener beforeNextTryListener = status -> {};
+  private RetryListener onSuccessListener = status -> {};
+  private RetryListener onFailureListener = status -> {};
+  private RetryListener onCompletionListener = status -> {};
+  private RetryConfig retryConfig = new RetryConfigBuilder().fixedBackoff5Tries10Sec().build();
+  private ExecutorService executorService = Executors.newSingleThreadExecutor();
+
+  @Test
+  public void shouldBuildCallExecutorWithConfigAndListeners(){
+    CallExecutor callExecutor = new CallExecutorBuilder()
+        .config(retryConfig)
+        .afterFailedTryListener(afterFailedTryListener)
+        .beforeNextTryListener(beforeNextTryListener)
+        .onSuccessListener(onSuccessListener)
+        .onFailureListener(onFailureListener)
+        .onCompletionListener(onCompletionListener)
+        .build();
+
+    assertEquals(retryConfig, callExecutor.getConfig());
```

Comment:
i only use AssertJ as the assertions library for this project and i'd prefer to stay consistent if possible, could you use `assertThat(retryConfig).isEqualTo(callExecutor.getConfig());` instead and assertj for the other assertions in this class as well?